analyzere/base_resources.py

```diff
@@ -243,7 +243,7 @@ def upload_data(self, file_or_str):
 
         # Block until data has finished processing
         while True:
-            resp = self.status
+            resp = self.processing_status
```

Comment:
Is it necessary to change this? We're no more at risk of name collisions than we were before. I think this change could be delayed until we adopt new-style uploads.